{{cookiecutter.project_slug}}/docker-compose.yaml

```diff
@@ -19,11 +20,13 @@ services:
       - local_postgres_data_backups:/backups
 
   redis:
+    container_name: {{cookiecutter.project_slug}}_redis
```

Comment:
I don't know that this is necessary, and may actually be less convenient than what we currently have.

For instance, I am used to running `docker compose up -d postgres redis` in my project directories, and docker compose automatically prefixes those containers with the directory name, which is also the project name. So they don't conflict with other projects (the port numbers conflict though).

Reply:
yea I was adding this mostly because I saw it added to the client and it was causing me an issue because I already had the client one running so I can just remove them all in that case!